listenbrainz_spark/stats/user/entity.py

```diff
@@ -129,6 +129,11 @@ def create_messages(data, entity: str, stats_range: str, from_ts: int, to_ts: in
     """
     for entry in data:
         _dict = entry.asDict(recursive=True)
+
+        # Clip the recordings to top 1000 so that we don't drop messages
+        if entity == "recordings":
```

Comment:
Could we only do this for all time? Because that's what is causing problems rn?

Reply:
Changed